src/catala/surface/lexer_fr.ml

```diff
@@ -565,7 +553,21 @@ let lex_law_fr (lexbuf : lexbuf) : token =
       done;
 
       LAW_ARTICLE (title, article_id, article_expiration_date)
-  | Plus (Compl ('@' | '/')) -> LAW_TEXT (Utf8.lexeme lexbuf)
+  | Plus '#', Star white_space, Plus (Compl ('[' | ']' | '\n')), Star white_space, '\n' ->
+      let extract_code_title = R.regexp "([#]+)[' ']*([^#\n]+)\n" in
+      let get_match = R.get_substring (R.exec ~rex:extract_code_title (Utf8.lexeme lexbuf)) in
+      let get_new_lines = R.regexp "\n" in
+      let new_lines_count =
+        try Array.length (R.extract ~rex:get_new_lines (Utf8.lexeme lexbuf)) with Not_found -> 0
+      in
+      for _i = 1 to new_lines_count do
+        new_line lexbuf
+      done;
+      let law_title = get_match 2 in
+      let precedence = String.length (get_match 1) - 1 in
+
+      LAW_HEADING (law_title, precedence)
+  | Plus (Compl ('@' | '/' | '#' | '`' | '>')) -> LAW_TEXT (Utf8.lexeme lexbuf)
```

Comment:
Oh ok I see, you have to know when to stop so you're forced to include `>` in the list. I guess it's fine, if need be we can invent a cleverer system to allow `>` in the text of law. Bonus: you can remove `@` from the list :)

Reply:
Before I can do it, I need to modify the syntax of law includes and master files.